tests/test_agent_runner.py

```diff
@@ -262,7 +263,7 @@ async def test_handoff_filters():
 
 
 @pytest.mark.asyncio
-async def test_async_input_filter_fails():
+async def test_async_input_filter_supported():
```

Comment:
if we didn't intentionally block async functions, i believe  changing this is a good addition